### PR TITLE
chore(server): clean up throttler

### PR DESCRIPTION
**Pull request by forehalo**
chore(server): clean up throttler

refactor(server): throttler

### CODE REVIEW

**Comment by forehalo** [comment]
* **#6326** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/6326?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @forehalo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/backend/server/tests/nestjs/throttler.spec.ts [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+  t.is(headers.limit, '120');
+  t.is(headers.remaining, '119');
+  t.is(headers.reset, '60');
+});
+
+test('should skip throttler for authenticated user when specified', async t => {
+  const { app, cookie } = t.context;
+
+  const res = await request(app.getHttpServer())
+    .get('/throttled/skip')
```
Comment (darkskygit) [inline packages/backend/server/tests/nestjs/throttler.spec.ts:284]: what about the unauthenticated user with skip decorator in throttled resolver?

Comment (forehalo) [reply packages/backend/server/tests/nestjs/throttler.spec.ts:284]: L150

Comment (darkskygit) [reply packages/backend/server/tests/nestjs/throttler.spec.ts:284]: L150 just tested without @ Throttle()

Comment (forehalo) [reply packages/backend/server/tests/nestjs/throttler.spec.ts:284]: L164

**Review by darkskygit — APPROVED** [review]

**Pull request merged**